_projects/2018/astropy/mcmc.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Add seamlessly compatibility between astropy.modeling.models and emcee
+desc: Implement a protocol to enable astropy.modeling.models to be used in emcee
+requirements:
+ - Python
+ - Solid understanding of probability (likelihood functions, Bayes' theorem, etc)
+ - Working knowledge on Markov Chain Monte Carlo
+difficulty: medium
+issues:
+ -
+mentors:
+ - mirca
+ - pllim
```

Comment:
I wonder whether Dan would be interested in being a mentor for this, too?

Reply:
But I have no experience with `emcee`. :scream: 

c/c @drdavella

Reply:
@pllim - not that Dan, but @dfm

Reply:
Ops, my bad. Sorry, Dan!